packages/terra-progress-bar/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Changelog
 Unreleased
 ----------
 
+2.3.0 - (March 26, 2018)
+------------------
+### Changed
```

Comment:
move these changes to be under unreleased

Reply:
Updated: https://github.com/cerner/terra-core/pull/1366/commits/6cc9d848c9f6c41e2f2848c2c07c613771a14e80